DIS.html

```diff
@@ -109,25 +92,56 @@ <h3>Section Three - Preparing your SD Card</h3>
+            <li><code><a href="magnet:?xt=urn:btih:3b59dd43eec3edb133555f58d1180bfb196acbb4&dn=2.54-0_11.4_NEW.firm&tr=udp%3A%2F%2Ftracker.coppersurfer.tk%3A6969%2Fannounce&tr=udp%3A%2F%2Ftracker.opentrackr.org%3A1337%2Fannounce&tr=http%3A%2F%2Ftracker.opentrackr.org%3A1337%2Fannounce&tr=udp%3A%2F%2Fzer0day.ch%3A1337%2Fannounce&tr=udp%3A%2F%2Ftracker.leechers-paradise.org%3A6969%2Fannounce&tr=udp%3A%2F%2Fexplodie.org%3A6969%2Fannounce&tr=http%3A%2F%2Fexplodie.org%3A6969%2Fannounce&tr=udp%3A%2F%2F9.rarbg.com%3A2710%2Fannounce&tr=udp%3A%2F%2Fp4p.arenabg.com%3A1337%2Fannounce&tr=http%3A%2F%2Fp4p.arenabg.com%3A1337%2Fannounce&tr=udp%3A%2F%2Ftracker.aletorrenty.pl%3A2710%2Fannounce&tr=http%3A%2F%2Ftracker.aletorrenty.pl%3A2710%2Fannounce&tr=http%3A%2F%2Ftracker1.wasabii.com.tw%3A6969%2Fannounce&tr=http%3A%2F%2Ftracker.baravik.org%3A6970%2Fannounce&tr=http%3A%2F%2Ftracker.tfile.me%2Fannounce&tr=udp%3A%2F%2Ftorrent.gresille.org%3A80%2Fannounce&tr=http%3A%2F%2Ftorrent.gresille.org%2Fannounce&tr=udp%3A%2F%2Ftracker.yoshi210.com%3A6969%2Fannounce&tr=udp%3A%2F%2Ftracker.tiny-vps.com%3A6969%2Fannounce&tr=udp%3A%2F%2Ftracker.filetracker.pl%3A8089%2Fannounce">2.54-0_11.4_NEW.firm</a></code></li>
+        </ul>
         <li>Copy the <code>boot.firm</code> from the Luma3DS 7z archive to the root of your SD Card</li>
         <li>Copy the <code>boot.3dsx</code> (Homebrew Menu) to the root of your SD Card</li>
         <li>Copy the <code>boot.nds</code> from the b9stool archive to the root of your SD Card</li>
         <li>Copy the <code>boot9strap.firm</code> from the <code>boot9strap.zip</code> file to the
             <code>/boot9strap/</code> folder on your SD Card. (Create it if it does not exist)
         </li>
+        <li>
+            Copy the <code>2.54-0_11.4_OLD.firm</code> or <code>2.54-0_11.4_NEW.firm</code> into the
+            <code>/boot9strap/</code> folder on your SD Card
+        </li>
+    </ol>
+    
+
+    <h3>Section Four - Installing boot9strap and configuring Luma</h3>
+    <ol>
+        <li>Launch b9stool by starting your DSiWare game</li>
+        <ul type=circle>
+            <li>You may need to load further into the game so it loads the modified save file</li>
+        </ul>
+        <li>Select "Dump F0F1" to backup the 3DS FIRM</li>
+        <li>Select "Install boot9strap" and confirm</li>
+        <li>Wait for it to say "done!". Exit b9stool and power off your device</li>
+        <ul type=circle>
+            <li>You may need to force power off by holding the power button</li>
+        </ul>
+        <li>Boot up your device to launch the Luma configuration menu</li>
+        <ul type=circle>
+            <li>
+                If your 3ds boots normally to the home menu then boot9strap has not installed successfully<br>
+                Replace your prep files with freshly downloaded files and restore the F0F1 backup in b9stool before attempting to install boot9strap again

```

Comment:
I am still siding with zoogie here - a F0F1 backup is not necessesary. Also add a note to try holding select on boot, luma may still pop and and it may have worked - i had one such case yesterday.

Reply:
If b9stool fails to install even have success for whatever reason (and this does happen, a lot), restoring F0F1 backups seems to have a better chance on reinstalling. The only reason Luma config screen shouldn't pop up (other than a fail install) is if someone copied a config.bin in luma folder from another hacked 3ds to their SD card. This is usually the case with system transfer dsiware because of being instructed to copy all files from source 3ds to target 3ds, but shouldn't be the case here.